components/map/layers/marker.md

```diff
@@ -12,7 +12,7 @@ position: 5
 
 The marker functionality allows you to add points in the map. These points are defined by geographical position in the map and can show useful information to the user in a tooltip.
 
-## How to configure Marker Layer?
```

Comment:
Either rephrase the section heading to something like To configure ... or delete this sentence.